### PR TITLE
FE Cleanup

### PR DESCRIPTION
**Pull request by tomwieck**

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -15,27 +14,13 @@ function App() {
...
-                  Separated link
-                </NavDropdown.Item>
-              </NavDropdown>
             </Nav>
           </Navbar.Collapse>
         </Container>
       </Navbar>
 
       <h1>
-        <span>Compare the weather for the next 5 days</span>
```
Comment (deemyboy) [inline frontend/src/App.tsx]: we need this as the mobile fix where we change text due to a small screen using a pseudo after element needs it

see _media_queries.scss line 14 -32 deals with that

File: frontend/src/scss/_city_table.scss [hunk 2]
```diff
@@ -46,6 +43,10 @@
   }
 }
 
+.city-row__weather {
+  padding-bottom: 16px;
```
Comment (deemyboy) [inline frontend/src/scss/_city_table.scss]: 16px = 1rem - keep consistent units please, ta

File: frontend/src/components/city_table/city_table.tsx [hunk 3]
```diff
@@ -54,49 +67,24 @@ export const CityTable: React.FC = () => {
...
 
-  const [selected, setSelected] = useState<City>();
-  const [searchString, setSearchString] = useState("");
-
   const handleOnSelect = (item: City) => {
     setCityId(item.id);
     setSearchString("");
   };
 
-  const [rowOpen, setRowOpen] = useState<IIsOpen>({
```
Comment (deemyboy) [inline frontend/src/components/city_table/city_table.tsx:65]: why has this whole section been removed?

it's very important for running the whole application

ln 65-81

Comment (tomwieck) [reply frontend/src/components/city_table/city_table.tsx:65]: Hasn't been removed, just moved to line 29 to keep all the State together

Comment (deemyboy) [reply frontend/src/components/city_table/city_table.tsx:65]: oops - total schoolboy from me

looks good

File: frontend/public/favicon.ico
Comment (deemyboy) [inline frontend/public/favicon.ico:1]: nice favicon

**Comment by deemyboy** [comment]
looks good - working fine

**Review by deemyboy — APPROVED** [review]

**Pull request merged**